dbxio/core/settings.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+import attrs
+
+from dbxio.core.cloud import CloudProvider
+
+_DEFAULT_CLOUD_PROVIDER = CloudProvider.AZURE  # TODO: get provider from env or remove this default at all
+_SUPPORTED_CLOUD_PROVIDERS = [CloudProvider.AZURE, CloudProvider.NEBIUS_OVER_AZURE]
+
+CLOUD_PROVIDER_ENV_VAR = 'CLOUD_PROVIDER'
+
+
+def _cloud_provider_factory() -> CloudProvider:
+    return CloudProvider(os.getenv(CLOUD_PROVIDER_ENV_VAR, _DEFAULT_CLOUD_PROVIDER).lower())
+
+
+@attrs.define
+class Settings:
+    cloud_provider: CloudProvider = attrs.field(factory=_cloud_provider_factory)
```

Comment:
I thought about some kind of _global_ settings like `pydantic_settings`. But this is ok